src/pages/components/SecondaryBar.qml

```diff
@@ -26,75 +26,90 @@ Item {
     height: isPortrait ? Settings.toolbarLarge : Settings.toolbarSmall
     clip: opacity < 1.0
 
-    Row {
+    Column {
         width: parent.width
-        height: parent.height
-
-        Browser.TabButton {
-            id: addTabButton
-            width: iconWidth + horizontalOffset
-            horizontalOffset: root.horizontalOffset
-            label.text: "+"
-            onTapped: enterNewTabUrl()
-        }
 
-        Browser.ExpandingButton {
-            id: forwardButton
-            expandedWidth: iconWidth
-            icon.source: "image://theme/icon-m-forward"
-            active: webView.canGoForward
-            onTapped: webView.goForward()
+        SecondaryBarItem {
```

Comment:
Maybe OverlayListItem instead. The "Secondary" part of above is a bit misleading as this is kind of third row.